packages/taquito-utils/src/errors.ts

```diff
@@ -1,3 +1,5 @@
+export { InvalidHexStringError, InvalidContractAddressError } from '@taquito/core';
```

Comment:
Do we also want to export InvalidAddressError, InvalidBlockHashError here to avoid breaking changes?